lib/matplotlib/tests/test_axes.py

```diff
@@ -1517,6 +1517,22 @@ def test_pcolorargs():
         ax.pcolormesh(X, Y, Z, shading='auto')
 
 
+def test_pcolormesh_underflow_error():
+    """
+    Test that underflow errors don't crop up in pcolormesh.  Probably
+    a numpy bug, but...
```

Comment:
```suggestion
    a numpy bug. See https://github.com/matplotlib/matplotlib/issues/27770.
```

Reply:
If we think this is a numpy bug, we should also report it upstream.